kimchi_description/launch/kimchi_description.launch.py

```diff
@@ -0,0 +1,70 @@
+# BSD 3-Clause License
```

Comment:
Mmmmmm, nuts, we have to fix licenses here and there. (Not asking to do it here)